releasenotes/notes/deferred_cluster_creation.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+
+releaseNotes:
+- |
+  **Added** an experimental feature to enable cluster creation on worker threads inline during requests.
+  This will save memory and CPU cycles in cases where there are lots of inactive clusters and > 1 worker thread.
+  This can be disabled by setting ENABLE_DEFERRED_CLUSTER_CREATION to true in agent Deployment.
```

Comment:
`ENABLE_DEFERRED_CLUSTER_CREATION` starts with `enable`, but `true` means `disable`? This confuses me.

Reply:
sorry. good point, That needs to change after the default has been changed. fixed now